test_regress/t/t_vpi_dump.iv.out

```diff
@@ -206,16 +134,6 @@ t (vpiModule) t
                 t.outer_scope[2].inner_scope[1].arr.LENGTH (vpiParameter) t.outer_scope[2].inner_scope[1].arr.LENGTH
                     vpiConstType=vpiBinaryConst
         t.outer_scope[2].inner_scope[2] (vpiGenScope) t.outer_scope[2].inner_scope[2]
-            vpiModule:
-            t.outer_scope[2].inner_scope[2].arr (vpiModule) t.outer_scope[2].inner_scope[2].arr
```

Comment:
This was repeated (vpiModule and vpiInternalScope), now just vpiInternalScope is needed.